src/state/queries/post-feed.ts

```diff
@@ -135,6 +142,15 @@ export function usePostFeedQuery(
   } | null>(null)
   const isDiscover = feedDesc.includes(DISCOVER_FEED_URI)
 
+  /**
+   * The number of posts to fetch in a single request. Because we filter
+   * unwanted content, we may over-fetch here to try and fill pages by
+   * `MIN_POSTS`.
+   */
+  const fetchLimit = React.useState(() => {
+    return gate('post_feed_lang_window') ? 100 : MIN_POSTS
+  })[0]
```

Comment:
this can just be 

```js
const fetchLimit = gate('post_feed_lang_window') ? 100 : MIN_POSTS
```

because gates are cached per-user in memory

Reply:
Ah right, thanks!